@app/db/scripts/toggle-admin.js

```diff
@@ -0,0 +1,28 @@
+const pg = require("pg");
+
+async function main() {
+  const pgPool = new pg.Pool({
+    host: process.env.DATABASE_HOST,
+    user: process.env.DATABASE_OWNER,
+    password: process.env.DATABASE_OWNER_PASSWORD,
+    database: process.env.DATABASE_NAME,
+  });
+  try {
+    await pgPool.query(
+      `update app_public.users set is_admin = $2 where username = $1;`,
+      [process.argv[2], process.argv[3]]
+    );
+    const res = await pgPool.query(
+      `select username, is_admin from app_public.users where username = $1;`,
+      [process.argv[2]]
+    );
+    console.log(res.rows);
```

Comment:
`console.table` might be a nice choice here. I leave that decision to you; leave it as-is if you want :)

```
> console.table({username: "Benjie", is_admin: true})
┌──────────┬──────────┐
│ (index)  │  Values  │
├──────────┼──────────┤
│ username │ 'Benjie' │
│ is_admin │   true   │
└──────────┴──────────┘
undefined
> console.table([{username: "Benjie", is_admin: true}])
┌─────────┬──────────┬──────────┐
│ (index) │ username │ is_admin │
├─────────┼──────────┼──────────┤
│    0    │ 'Benjie' │   true   │
└─────────┴──────────┴──────────┘
undefined
```